libraries/Gamebuino-Meta/src/utility/Sound/Sound_FX.cpp

```diff
@@ -0,0 +1,154 @@
+/*
+This file is part of the Gamebuino-Meta library,
+Copyright (c) Aadalie 2017
```

Comment:
it's 2018! Also, it says (c) Aadalie, which is the gamebuino company, if you want to give over copyright, sure, it is up to you, though